Models/API.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  API.swift
+//  iosdevapp
+//
+//  Created by Leonit Ejupi on 02.03.23.
+//
+
+import Foundation
+
+enum JSONFileError : Error{
+    case JSONFileNotFound
+}
+
+func fileLoaderService(fileName: String) throws -> APISecretData {
```

Comment:
I would wrap this function in a `class` called `FileLoaderService` and give it a more descriptive name like `loadAPISecretFromJSONFile`.